test/protect.jl

```diff
@@ -16,17 +16,53 @@
     end
 
     @testset "unprotect" begin
+        using StatsModels: TupleTerm, FullRank
         # unprotect reverts to treating calls to +, &, and * as term union,
         # interaction, and combined
 
+        # helper to "lift" function into FunctionTerm constructor
+        function ft(f)
+            return function(args...)
+                return FunctionTerm(f, args, :($f($(args...))))
+            end
+        end
```

Comment:
Why not define this simply as:
```suggestion
    ft(f, args...) = FunctionTerm(f, args, :($f($(args...))))
```
and call it as `ft(op, a, b)` instead of `ft(op)(a, b)`?

Reply:
yeah that works too!  I think the version here was inspired by the "lift" model of "passmissing" and friends.  But for tests I'm kind of indifferent.